twisted/web/http.py

```diff
@@ -2251,15 +2251,15 @@ class _GenericHTTPChannelProtocol(proxyForInterface(IProtocol, "_channel")):
     @property
     def factory(self):
         """
-        @see: L{HTTPChannel.factory}
+        @see: L{_genericHTTPChannelProtocolFactory}
         """
         return self._channel.factory
 
 
     @factory.setter
     def factory(self, value):
         """
-        @see: L{HTTPChannel.factory}
+        @see: L{_genericHTTPChannelProtocolFactory}
```

Comment:
this ok, but for setter we should not need docstring.

the API doc only exports the getter docstring